lib/bike_brigade_web/live/rider_live/form_component.ex

```diff
@@ -154,6 +154,7 @@ defmodule BikeBrigadeWeb.RiderLive.FormComponent do
     save_rider_edit_impl(socket, rider_form_params)
   end
 
+  # TODO: this can probably be deleted: we don't really create riders from the form?
```

Comment:
```suggestion
  # TODO: this can probably be deleted: we don't really create riders from the form?
  # https://github.com/bikebrigade/dispatch/issues/321
```